brd-ios/breadwallet/src/ViewControllers/RootNavigationController.swift

```diff
@@ -62,7 +61,8 @@ class RootNavigationController: UINavigationController, UINavigationControllerDe
             is VerifyAccountViewController,
             is SwapInfoViewController,
             is OrderPreviewViewController,
-            is BaseInfoViewController:
+            is BaseInfoViewController,
+            is ItemSelectionViewController:
```

Comment:
r u sure this is not breaking any other flows? 🤔

Reply:
I think so yes, without this we have clear navigation on search bar in item selection..

https://user-images.githubusercontent.com/40755142/199699017-e2e6aabd-e7d8-4fb6-8c9b-d55156e6b076.mov